packages/flutter/lib/src/cupertino/refresh.dart

```diff
@@ -359,43 +357,53 @@ class CupertinoSliverRefreshControl extends StatefulWidget {
     return state.refreshState;
   }
 
-  /// Builds a simple refresh indicator that fades in a bottom aligned down
-  /// arrow before the refresh is triggered, a [CupertinoActivityIndicator]
-  /// during the refresh and fades the [CupertinoActivityIndicator] away when
-  /// the refresh is done.
-  static Widget buildSimpleRefreshIndicator(
+  /// Builds a refresh indicator that reflects the standard iOS pull-to-refresh
+  /// behaviour. Specifically, this entails presenting an activity indicator that
+  /// changes depending on the current refreshState. As the user initially drags
+  /// down, the indicator will gradually reveal individual ticks until the refresh
+  /// becomes armed. At this point, the animated activity indicator will begin rotating.
```

Comment:
Nit: Would it help to include references to the states like `[RefreshIndicatorMode.armed]` here?

Reply:
I had a couple of goes at trying this, but each attempt resulted in an unwieldy mouthful. 

At the moment, the function description is more like an overview, rather than an explanation of the internal mechanics. As a consumer of this function, I'm not sure if I would really need the gory details, which seem reasonably clear in the `switch` statement further down, but I'm happy to try further if you feel it is warranted.

Reply:
No worries, I'm happy with leaving it as-is.  Thanks for taking a look.